Tests/RegexTests/MatchTests.swift

```diff
@@ -246,110 +256,155 @@ extension RegexTests {
 
     // MARK: Character names.
 
-    matchTest(#"\N{ASTERISK}"#, input: "123***xyz", match: "*", xfail: true)
-    matchTest(#"[\N{ASTERISK}]"#, input: "123***xyz", match: "*", xfail: true)
+    matchTest(#"\N{ASTERISK}"#, input: "123***xyz", match: "*")
+    matchTest(#"[\N{ASTERISK}]"#, input: "123***xyz", match: "*")
     matchTest(
-      #"\N{ASTERISK}+"#, input: "123***xyz", match: "***", xfail: true)
+      #"\N{ASTERISK}+"#, input: "123***xyz", match: "***")
     matchTest(
-      #"\N {2}"#, input: "123  xyz", match: "3  ", xfail: true)
+      #"\N {2}"#, input: "123  xyz", match: "3  ")
 
     matchTest(#"\N{U+2C}"#, input: "123,xyz", match: ",")
     matchTest(#"\N{U+1F4BF}"#, input: "123💿xyz", match: "💿")
     matchTest(#"\N{U+00001F4BF}"#, input: "123💿xyz", match: "💿")
 
     // MARK: Character properties.
 
-    matchTest(#"\p{L}"#, input: "123abcXYZ", match: "a", xfail: true)
-    matchTest(#"\p{gc=L}"#, input: "123abcXYZ", match: "a", xfail: true)
-    matchTest(#"\p{Lu}"#, input: "123abcXYZ", match: "X", xfail: true)
-    matchTest(#"\P{Cc}"#, input: "123\n\n\nXYZ", match: "\n", xfail: true)
-    matchTest(#"\P{Z}"#, input: "123 XYZ", match: " ", xfail: true)
+    matchTest(#"\p{L}"#, input: "123abcXYZ", match: "a")
+    matchTest(#"\p{gc=L}"#, input: "123abcXYZ", match: "a")
+    matchTest(#"\p{Lu}"#, input: "123abcXYZ", match: "X")
+
+    // TODO: diagnose
+    matchTest(
+      #"\P{Cc}"#, input: "123\n\n\nXYZ", match: "\n",
+      xfail: true)
+    matchTest(
+      #"\P{Z}"#, input: "123 XYZ", match: " ",
+      xfail: true)
```

Comment:
When I was writing these I think I missed that capital P was a negation. The following should work:

```suggestion
    // TODO: diagnose
    matchTest(
      #"\P{Cc}"#, input: "\n\n\nXYZ", match: "X")
    matchTest(
      #"\P{Z}"#, input: "   XYZ", match: "X")
```